Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+		public static void Demo()
+		{
+			/*2) Take homework 2 code, place it in the forked branch and refactor it using functions.There should be as little duplicate code as possible (there should be functions for:
+				-Calculating BMI(weight comes in kg, height comes in meters),
+			-Prompt for input and converting it to int(print message for request, read console input and return converted input to int), 
+			-Prompt for input and converting it to string(print message for request, read console input and return input),
+			-Prompt for input and converting it to float(print message for request, read console input and return converted input to float).
+			3) Put all the function you made into the right places in Checks.cs class. Run tests, make sure you pass all the tests
+			4) Put all program.cs logic to Lesson3.cs.Call it from main function. Program class should look like this:
+			*/
+
+			string quit = "";
+
+			while (quit != "q" && quit != "Q")
+			{
+				int age = getUserInt("age");
```

Comment:
Use a function that tells what is being done. You need to process person info, might as well call the contents of the loop like that.

Reply:
refactored per instruction.